<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAndReplaceWidget</class>
 <widget class="QWidget" name="FindAndReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find And Replace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameHeader">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelIcon">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap resource="../icons/icons.qrc">:/icons/16x16/find-and-replace.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Find and Replace</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkRegularExpression">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Interpret text to find as regular expression and allow placeholders in replacement text</string>
        </property>
        <property name="text">
         <string>Regular Expression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkWholeWord">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Find only whole words</string>
        </property>
        <property name="text">
         <string>Whole Word</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkCaseSensitive">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Respect case of text to find</string>
        </property>
        <property name="text">
         <string>Case Sensitive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="buttonHide">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Hide</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboFind">
     <property name="toolTip">
      <string>Text to find</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboReplace">
     <property name="toolTip">
      <string>Replacement text</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonPrevious">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReplace">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReplaceAndNext">
       <property name="text">
        <string>Replace &amp;&amp; Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReplaceAll">
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listResults"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboFind</tabstop>
  <tabstop>comboReplace</tabstop>
  <tabstop>buttonPrevious</tabstop>
  <tabstop>buttonNext</tabstop>
  <tabstop>buttonReplace</tabstop>
  <tabstop>buttonReplaceAndNext</tabstop>
  <tabstop>buttonReplaceAll</tabstop>
  <tabstop>listResults</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
